### DISCUSSION TITLE
Additional Clock Synced LFO Time Divisions

### DISCUSSION THREAD

**Discussion by Charlie-Brohawn** [Pirate-MIDI/Pirate-MIDI-Features-Bug-Tracking#87] [Feature Requests & Ideas]
While the LFOs can currently be set to pretty low frequencies in free sync mode, it would be nice to to have clock synced LFO frequencies that extended to the length of multiple measures. Half notes, dotted and triplet values extending to say, dotted whole notes, then natural number multiples of the bar length up to 8 measures. There’s something attractive from a musical standpoint about the imprecision of free running LFOs and the way they phase in and out of the planned tempo of a performance, but there are times when syncing slow yet relatively continuous movement to a precise tempo is preferred.

**Comment by imranino** [comment]
Yes! This! The LFOs are truly amazing, but the parameters are a bit too limited. Much slower oscillations would be amazing, and dotted/triplet subdivisions would be fun as well.
